### DISCUSSION TITLE
posts/vim-tutor-for-oi

### DISCUSSION THREAD

**Discussion by giscus** [EarthMessenger/blog.earthmessenger.xyz#9] [giscus]
# 2022/08/23/vim-tutor-for-oi

缺点：同学没法帮你调代码了

https://blog.earthmessenger.xyz/2022/08/23/vim-tutor-for-oi.html

**Comment by fat-old-eight** [comment]
~《OIer 的 VIM 教程》~
《OIer 的 VIM 劝退教程》

**Comment by Rickyxrc** [comment]
十分有用，感谢大佬！
